src/test/java/harness/utils/WorkspaceUtils.java

```diff
@@ -23,12 +24,16 @@ public static String createUserFacingId() {
    * @param workspaceCreator The user who owns the workspace. This user will be impersonated to in
    *     the WSM workspaceDelete request.
    */
-  public static UFWorkspace createWorkspace(TestUser workspaceCreator)
+  public static UFWorkspace createWorkspace(TestUser workspaceCreator, CloudPlatform cloudPlatform)
```

Comment:
Hmm the point of these helper methods is to get rid of boilerplate. Now we have `CloudPlatform.GCP` in a bunch of places. What do you think:

Rename this to createGcpWorkspace. Either hard-code to GCP, or omit (will default to GCP).

Reply:
This change was a quick change until we add tests for AWS, at which time the same functions could have been used with multiple clouds. 

however with addition of GCP as default value, this entire change in tests can be removed